docs/section22/Rule22-12.md

```diff
@@ -0,0 +1,65 @@
+
+1. get_heat_rejection_loops_connected_to_baseline_systems()
+
+**Applicability Checks:**  
+
+- The function get_heat_rejection_loops_connected_to_baseline_systems() returns a list of loops that are connected to Type 7,8,12,13,7b,8b,12b.  Get the list of applicable heat rejection loops: `heat_rejection_loop_ids = get_heat_rejection_loops_connected_to_baseline_systems(B_RMI)`
+
+- check applicability for each loop in the model: `for fluid_loop in B_RMI.fluid_loops:`
+
+  - check if heat_rejection is one of the applicable loops: `if fluid_loop.id in heat_rejection_loop_ids: CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to the heat rejection: `else: RULE_NOT_APPLICABLE`
+
+  ## Rule Logic:  
+
+  - create a heat_rejections_count variable to count the number of heat_rejections that are connected to the fluid_loop: `heat_rejections_count = 0`
+
+  - loop through the heat rejections in the model: `for heat_rejection in B_RMI.heat_rejections:`
+
+    - check if the heat_rejection is connected to the fluid_loop, and increment the heat_rejections_count if it is: `if heat_rejection.fluid == fluid_loop: heat_rejections_count += 1`
+
+  - check whether there is more than one heat rejection loop for the chiller plant.  To do this, we need to find a chiller that is connected to the heat rejection loop.  Look at all chillers: `for chiller B_RMI.chillers:`
+  
+    - check if the chiller is connected to the heat rejection loop, and if so, break out of the loop: `if chiller.condensing_loop == fluid_loop: break`
+
+  - create variable one_heat_rejection_loop to keep track of whether there are one or more heat rejection loops connected to the chiller plant.  Set this variable equal to true (at the moment, we know of only one heat rejection loop): `one_heat_rejection_loop = true`
+
+  - find all the other chillers connected to the same cooling loop: `for other_chiller in B_RMI.chillers:`
+
+    - check if other_chiller is connected to the same cooling loop: `if other_chiller.cooling_loop = chiller.cooling_loop :`
+
+      - check if the other_chiller is not connected to the same condensing_loop: `if other_chiller.condensing_loop != fluid_loop:`
+
+        - this means that there is more than one fluid loop for the chilled water plant, set one_heat_rejection_loop equal to false and break out of the loop: `one_heat_rejection_loop = false; break`
+
+
+  **Rule Assertion - FluidLoop:**
+
+  - Case 1: If there is exactly one heat rejection connected to the fluid loop, AND there is exactly one heat rejection loop, PASS: `if heat_rejections_count == 1 AND one_heat_rejection_loop: PASS`
+
+  - Case 2: Elsif there is exactly one heat rejection connected to the fluid loop, but there is more than one fluid loop, fail and include note: `elsif heat_rejections_count == 1 AND !one_heat_rejection_loop: FAIL; note = "there is more than one condensing loop for this chiller plant.  There should only be one condensing loop attached to all chillers in the chiller plant"`
```

Comment:
done

Reply:
@KarenWGard i dont see the change of 'cooling tower' instead of 'heat rejection' and 'condenser loop' instead of 'condensing loop'. Not sure if I'm doing something wrong on GitHub